app/src/main/jni/Application.mk

```diff
@@ -0,0 +1,6 @@
+APP_OPTIM := debug
```

Comment:
@tladesignz this should be changed to `release` once we're happy with the implementation and ready to merge...